app/models/user.rb

```diff
@@ -9,4 +9,6 @@ class User < ApplicationRecord
   has_many :managed_projects, class_name: "Project"
   has_many :user_projects
   has_many :contributed_projects, through: :user_projects, source: :project
+
+  has_many :authored_events, class_name: "Event", foreign_key: "author_id"
```

Comment:
I wanted to make this association more descriptive, like `contributed_projects`.